src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -116,7 +114,7 @@ const loadPage = async(
   bookLoader: ReturnType<AppServices['archiveLoader']['book']>,
   pageId: string
 ) => {
-  services.dispatch(requestPage(match.params.page));
+  services.dispatch(requestPage({slug: match.params.page.slug}));
```

Comment:
shouldn't this be 
```suggestion
  services.dispatch(requestPage(match.params.page));
```

Reply:
I've done it expecting that eventually routing for pages will be similar to routing for books, i.e pages can be requested with id.